src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private FigureList object;
+    private Figure randomFigure;

```

Comment:
same, lets make this variable local

Reply:
what about this comment?

Reply:
When this variable is local, program don't work